src/sentry/grouping/enhancer/actions.py

```diff
@@ -142,7 +141,6 @@ class VarAction(Action):
     _VALUE_PARSERS: dict[str, Callable[[Any], Any]] = {
         "max-frames": int,
         "min-frames": int,
-        "invert-stacktrace": get_rule_bool,
```

Comment:
Originally added in https://github.com/getsentry/sentry/pull/24346/files#diff-9f2f24a63332eb6faec958923e3483274496c40c97528f0da431fb663d2f7d58R375